react/hooks/useGetBagsSgrIDs.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react"
+
+const useGetBagsSgrIDs = () => {
+  const [bagsIDs, setBagsIDs] = useState([''])
+  const [sgrIDs, setSgrIDs] = useState([''])
+
+  useEffect(() => {
+    const getSettings = () => {
```

Comment:
Pentru ca ai fetch in acest useEffect, ar trebui sa folosesti o logica care sa verifice daca nu cumva componenta ta a fost unmounted intre timp, inainte sa faci setState.

Ex:

  useEffect(() => {

    let isSubscribed = true

    const fetchFunc = () => {

      fetch('url')
        .then(res => res.json())
        .then(result => {

          if (isSubscribed) {
              setState(result)
          }
        })
    }

    return () => {
      isSubscribed = false
    }
  }, [])

Reply:
am facut cu useref